code/modules/mob/living/carbon/alien/humanoid/alien_powers.dm

```diff
@@ -287,7 +287,7 @@ Doesn't work on other aliens/AI.*/
 	spell_flags = IGNORESPACE|NODUPLICATE
 
 	summon_type = list(/obj/effect/alien/egg)
-a
```

Comment:
How did this compile

Reply:
It's valid BYOND code
It's a type definition
If you compile the code currently in Dream Maker and look at the object tree, you'll see a nice little empty node named `a` hanging out in there